keystore/utils.go

```diff
@@ -49,3 +49,56 @@ func PrintKeysTable(keys []KeyDescription, writer io.Writer) error {
 	}
 	return nil
 }
+
+// PrintHistoricalKeysTable prints table which describes keys in a human readable format
+// into the writer.
+// Code is shared by `acra-keys list` and a couple of tests
+func PrintHistoricalKeysTable(keys []KeyDescription, writer io.Writer) error {
```

Comment:
can you add example of the expected output here? it simplifies understanding what the code does and what is expected